w02/src/academy/pocu/comp2500samples/w02/vehicle/Vehicle.java

```diff
@@ -0,0 +1,81 @@
+package academy.pocu.comp2500samples.w02.vehicle;
+
+import java.util.ArrayList;
+
+public class Vehicle {
+    public VehicleType Type;

```

Comment:
all member variables should be small case no? (i know they are currently public member variables atm, but ... it's just midstep to go to private member variables soon.. so i'd like to keep them as lower)